notebooks/datasets/volcano-so2-monitoring.ipynb

```diff
@@ -701,6 +733,13 @@
     "\n",
     "In this case study we have successfully visualized how NASA monitors sulfur dioxide emissions from space, by showcasing three different examples across the globe: volcanic activity in the Galápagos islands (2005), large scale emissions on the Kamchatka Peninsula (2009), and eruptions of Fagradalsfjall in Iceland (2021)."
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
```

Comment:
```suggestion
```

Reply:
I _think_ this will remove the empty cell

Reply:
I'm going to save this for a new PR (maybe in the next?)  I accidentally included these files in another PR and it will be way easier to resolve that if this gets merged as is